cql/operation/fhir-operation-cqf/src/main/java/com/ibm/fhir/operation/cqf/MeasureCollectDataOperation.java

```diff
@@ -108,7 +108,7 @@ protected void resolveReferences(Resource resource, Parameters.Builder parameter
 
                         if( ! resourceMap.containsKey(id) ) {
                             try {
-                                SingleResourceResult<? extends Resource> readResult = resourceHelper.doRead(type, id, /*throwExOnNull=*/true, /*includeDeleted=*/false, /*contextResource=*/null);
+                                SingleResourceResult<? extends Resource> readResult = resourceHelper.doRead(type, id, /*throwExOnNull=*/true, /*contextResource=*/null);
```

Comment:
Didn't you get rid of contextResource?

Reply:
fixed by removing the trailing param.  found one other case like this and fixed that too.  if it were any value other than null it would be a compile error.

Reply:
did a search for "contextResource" to confirm I got them all now